azurerm/internal/services/devtestlabs/tests/resource_arm_dev_test_global_shutdown_schedule_test.go

```diff
@@ -0,0 +1,243 @@
+					resource.TestCheckResourceAttr(data.ResourceName, "tags.%", "1"),
+					resource.TestCheckResourceAttr(data.ResourceName, "tags.environment", "Production"),
+				),
+			},
+		},
+	})
+}
+
+func testCheckAzureRMDevTestLabGlobalShutdownScheduleExists(resourceName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[resourceName]
+		if !ok {
+			return fmt.Errorf("Not found: %s", resourceName)
+		}
+
+		targetResourceID := rs.Primary.Attributes["target_resource_id"]
+		exists, err := testCheckAzureRMDevTestLabGlobalShutdownScheduleExistsInternal(targetResourceID)
+
+		if err != nil {
+			return fmt.Errorf("Error checking if item has been created: %s", err)
+		}
+		if !exists {
+			return fmt.Errorf("Bad: Dev Test Lab Global Schedule %q does not exist", targetResourceID)
+		}
+
+		return nil
+	}
+}
+
+func testCheckAzureRMDevTestLabGlobalShutdownScheduleDestroy(s *terraform.State) error {
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "azurerm_dev_test_global_shutdown_schedule" {
+			continue
+		}
+
+		targetResourceID := rs.Primary.Attributes["target_resource_id"]
+		exists, err := testCheckAzureRMDevTestLabGlobalShutdownScheduleExistsInternal(targetResourceID)
+
+		if err != nil {
+			return fmt.Errorf("Error checking if item has been destroyed: %s", err)
+		}
+		if exists {
+			return fmt.Errorf("Bad: Dev Test Lab Global Schedule %q still exists", targetResourceID)
+		}
+	}
+
+	return nil
+}
+
+func testCheckAzureRMDevTestLabGlobalShutdownScheduleExistsInternal(vmID string) (bool, error) {
+	client := acceptance.AzureProvider.Meta().(*clients.Client).DevTestLabs.GlobalLabSchedulesClient
+	ctx := acceptance.AzureProvider.Meta().(*clients.Client).StopContext
+
+	rid, err := azure.ParseAzureResourceID(vmID)
+	if err != nil {
+		return false, fmt.Errorf("Bad: Failed to parse ID (id: %s): %+v", vmID, err)
+	}
+
+	vmName := rid.Path["virtualMachines"]
+	name := "shutdown-computevm-" + vmName // Auto-shutdown schedule must use this naming format for Compute VMs
+	resourceGroup := rid.ResourceGroup
+
+	resp, err := client.Get(ctx, resourceGroup, name, "")
+	if err != nil {
+		if resp.Response.IsHTTPStatus(404) {
+			return false, nil
+		}
+		return false, fmt.Errorf("Bad: Get on devTestLabsGlobalSchedules client (id: %s): %+v", vmID, err)
+	}
+
+	return true, nil
+}
+
+func testAccAzureRMDevTestLabGlobalShutdownSchedule_template(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_virtual_network" "test" {
+  name                = "acctvn-%d"
+  address_space       = ["10.0.0.0/16"]
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+}
+
+resource "azurerm_subnet" "test" {
+  name                 = "acctsub-%d"
```

Comment:
all resource names should contain acctest and caps when possible
```suggestion
  name                 = "acctest-SN-%d"
```

Reply:
Makes sense. I updated all resources (where possible) to use that naming format.